source/iterators.tex

```diff
@@ -1885,7 +1885,7 @@
 \ref{bidirectional.iterators}, \ref{random.access.iterators}).
 
 \pnum
-A type \tcode{X} meets the \oldconcept{Iterator} requirements if:
+A type \tcode{X} meets the \defnoldconcept{Iterator} requirements if:
```

Comment:
It seems surprising to have two `defnoldconcept`s for this.

Reply:
fixed